src/python_testing/matter_testing_infrastructure/matter_testing_support/spec_parsing.py

```diff
@@ -518,19 +519,36 @@ class DataModelLevel(str, Enum):
     kDeviceType = 'device_types'
 
 
-def _get_data_model_directory(data_model_directory: typing.Union[PrebuiltDataModelDirectory, str], data_model_level: DataModelLevel) -> str:
+def _get_data_model_root() -> str:
```

Comment:
I don't think this is going to work in the test harness. If bundling the module to make it portable is going to break this pathing, then we need to consider packing the data model files into the built module.

Reply:
TH will probably have to figure this out in some way. Since it passes today, I imagine TH pulls in a full checkout and has XML files. We should probably also support some additional env variable (like MATTER_DATA_MODEL_PATH) that TH can use until we can bundle XMLs into the module itself.